packages/web3torrent/.storybook/web3torrent-client.mock.js

```diff
@@ -1,22 +1,19 @@
 const React = require('react');
 const utils = require('./../src/utils/test-utils');
 
-const mockWeb3TorrentClient = () => {
+function mockWeb3TorrentClient() {
   const _peers = utils.createMockTorrentPeers();
-  const torrent = utils.createMockTorrent({
+  const torrent = utils.createMockTorrentUI({
     numPeers: Object.keys(_peers).length,
     _peers,
     wires: Array.from(_peers, ({ wire }) => wire)
   });
 
-  return { torrents: [torrent] };
+  return { torrents: [torrent], paymentChannelClient: {challengeChannel:(id)=>{}} };
 };
 
 const web3Torrent = mockWeb3TorrentClient();
 
 const Web3TorrentContext = React.createContext(web3Torrent);
 
-module.exports = {
-  Web3TorrentContext,
-  web3Torrent
-};
+module.exports = {Web3TorrentContext,web3Torrent};
```

Comment:
Do you know if these files are linted? 
```suggestion
module.exports = {Web3TorrentContext, web3Torrent};
```